addons/sms/wizard/template_reset_wizard_views.xml

```diff
@@ -0,0 +1,31 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+    <record id="template_reset_wizard_view_form" model="ir.ui.view">
```

Comment:
Would make a dedicated small model in sms (sms.template.reset), that has a template_ids field. That way you have specific fields and code, and don't deal with active model / ids :) .